api/src/controllers/tickets.rs

```
@@ -148,7 +159,7 @@ pub fn redeem(
             },
         Err(e) => Ok(HttpResponse::new(StatusCode::INTERNAL_SERVER_ERROR)
```

We should really throw a better error here, doesn't need to be done in this ticket though